WindowsServerDocs/identity/ad-ds/get-started/virtual-dc/virtualized-domain-controllers-hyper-v.md

```diff
@@ -258,35 +258,36 @@ If a valid system state backup exists for the domain controller virtual machine,
-9. Right-click the **Directory Services** log, and then click **Find**. In **Find what**, type **1109**, and then click **Find Next**.
-10. You should see at least an Event ID 1109 entry. If you do not see this entry, proceed to the next step. Otherwise, double-click the entry, and then review the text confirming that the update was made to the InvocationID:
+1. Using the previous VHD, start the virtual domain controller in DSRM, as described in the previous section. Don't allow the domain controller to start in normal mode. If you miss the Windows Boot Manager screen and the domain controller begins to start in normal mode, turn off the virtual machine to prevent it from completing startup. See the previous section for detailed instructions for entering DSRM.
+1. Open Registry Editor. To open Registry Editor, select **Start**, select **Run**, type **regedit**, and then select OK. If the **User Account Control** dialog box appears, confirm that the action it displays is what you want, and then select **Yes**. In Registry Editor, expand the following path: **HKEY\_LOCAL\_MACHINE\\SYSTEM\\CurrentControlSet\\Services\\NTDS\\Parameters**. Look for a value named **DSA Previous Restore Count**. If the value is there, make a note of the setting. If the value isn't there, the setting is equal to the default, which is zero. Don't add a value if you don't see one there.
+1. Right-select the **Parameters** key, select **New**, and then select **DWORD (32-bit) Value**.
+1. Type the new name **Database restored from backup**, and then press ENTER.
+1. Double-select the value that you just created to open the **Edit DWORD (32-bit) Value** dialog box, and then type **1** in the **Value data** box. The **Database restored from backup entry** option is available on domain controllers that are running Windows 2000 Server with Service Pack 4 (SP4), Windows Server 2003 with the updates that are included in [How to detect and recover from a USN rollback in Windows Server 2003, Windows Server 2008, and Windows Server 2008 R2](/troubleshoot/windows-server/identity/detect-and-recover-from-usn-rollback) in the Microsoft Knowledge Base installed, and Windows Server 2008.
+1. Restart the domain controller in normal mode.
+1. When the domain controller restarts, open Event Viewer. To open Event Viewer, select **Start**, select **Control Panel**, double-select **Administrative Tools**, and then double-select **Event Viewer**.
+1. Expand **Application and Services Logs**, and then select the **Directory Services** log. Ensure that events appear in the details pane.
+1. Right-select the **Directory Services** log, and then select **Find**. In **Find what**, type **1109**, and then select **Find Next**.
+1. You should see at least an Event ID 1109 entry. If you don't see this entry, proceed to the next step. Otherwise, double-select the entry, and then review the text confirming that the update was made to the InvocationID:
```

Comment:
Throughout this file, "right-select" and "double-select" should be reverted to "right-click" and "double-click", per MS style.